packages/react-theming/src/examples/compose/index.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import { initalize } from './../..';
```

Comment:
make sure that file paths are explicit. This import would break an AMD build. (In fabric we have a build step that validates file path imports go to explicit files because of this nuance.)